src/Common/MoistThermodynamics/states.jl

```diff
@@ -143,17 +161,19 @@ Constructs a [`PhaseEquil`](@ref) thermodynamic state from:
  - `θ_liq_ice` liquid-ice potential temperature
  - `ρ` (moist-)air density
  - `q_tot` total specific humidity
- - `tol` tolerance for saturation adjustment
+ - `temperature_tol` estimated temperature tolerance for saturation adjustment
  - `maxiter` maximum iterations for saturation adjustment
 """
 function LiquidIcePotTempSHumEquil(
     param_set::APS,
     θ_liq_ice::FT,
     ρ::FT,
     q_tot::FT,
-    maxiter::Int = 30,
-    tol::FT = FT(1e-1),
+    maxiter::Int = 40,
```

Comment:
Seems much too high:
```suggestion
    maxiter::Int = 10,
```

Reply:
At `temperature_tol=1e-1`, the best we can do is `maxiter=36`

Reply:
This shouldn't be a problem since all constructors aside from
 - `PhaseDry`
 - `PhaseEquil`
 - `PhaseNonEquil`
are only used for initialization